packages/google_maps_flutter/google_maps_flutter_android/pigeons/messages.dart

```diff
@@ -46,14 +46,27 @@ class PlatformCameraUpdate {
 
 /// Pigeon equivalent of the Circle class.
 class PlatformCircle {
-  PlatformCircle(this.json);
+  PlatformCircle({
+    required this.circleId,
+    required this.center,
+    this.consumeTapEvents = false,
+    this.fillColor = 0x00000000,
+    this.strokeColor = 0xFF000000,
+    this.visible = true,
+    this.strokeWidth = 10,
+    this.zIndex = 0.0,
+    this.radius = 0,
```

Comment:
Optional: you can generally just make everything `required` in a Pigeon definition for a class like this that corresponds pretty directly to an existing class, because in practice there are usual no calls that don't provide every field due to the nature of how we use them.

Reply:
At least one case where they were not all provided is the Java unit tests prior to this PR. Technically there was only one field at that time, the `json`, but that json field only had the fields specified that were actually tested by the unit tests. If any of the non-null fields are not provided to the `Builder` nested class, `build` will throw an exception. I was hoping that providing default values would mean that the Builder would use them and not require them all to be specified, avoiding this problem, but it appears that the generated Java code does not use the defaults anyway.

Reply:
Yes, default values haven't been implemented for any of the just languages. Usually for unit tests we just make a helper that builds the object with mostly default values.

Reply:
@yaakovschectman did you and stuart agree not to make these values required?